.travis.yml

```diff
@@ -16,8 +16,10 @@ env:
 
 branches:
   only:
-    - develop
     - master
+    - /^v(0|[1-9]\d*)\.(0|[1-9]\d*)$/
```

Comment:
We don't need it anymore. It's obsolete naming.

Reply:
Done